lib/restivus.coffee

```diff
@@ -309,11 +311,19 @@ class @Restivus
       get: ->
         # Remove the given auth token from the user's account
         authToken = @request.headers['x-auth-token']
-        Meteor.users.update @user._id, {$pull: {'services.resume.loginTokens': {token: authToken}}}
+        hashedToken = Accounts._hashLoginToken authToken
+        tokenPaths = self.config.auth.token.split '.'
+        tokenPrefix = tokenPaths[..-2].join '.'
+        tokenSuffix = tokenPaths[-1..][0]
+        pullQueryTarget = {}
```

Comment:
I'm sure there are some small things that could be done to clean it up in the future, but for now it works, and it looks OK to me. I'd recommend a few variable name changes, for a bit more clarity:

`tokenPaths` -> `tokenLocation`
`tokenPrefix` -> `pathToTokens`
`tokenSuffix` -> `tokenFieldName`
`pullQueryTarget` -> `tokenToRemove`
`pullQuery` -> `tokenRemovalQuery`

I'm not in love with the last one, but it's more descriptive than `pullQuery`. If you think of anything better for any of them feel free to use that instead.